lib/connection.ts

```diff
@@ -452,19 +515,21 @@ export class Connection extends Entity {
 
   /**
    * Creates an amqp session on the provided amqp connection.
+   * @param abortSignal A signal used to cancel the operation.
    * @return {Promise<Session>} Promise<Session>
    * - **Resolves** the promise with the Session object when rhea emits the "session_open" event.
    * - **Rejects** the promise with an AmqpError when rhea emits the "session_close" event while
-   * trying to create an amqp session.
+   * trying to create an amqp session or with an AbortError if the operation was cancelled.
    */
-  createSession(): Promise<Session> {
+  createSession(abortSignal?: AbortSignalLike): Promise<Session> {
```

Comment:
Should be in `options` as well, like we have with the others?

Reply:
Yes, fixed in https://github.com/amqp/rhea-promise/pull/66/commits/2c9e83353a1d12d3665db5fe27c95ccd33d09fae